### PR TITLE
Replace ActiveRecord::Import#import for Rails 6 bulk insert

### PR DESCRIPTION
**Pull request by deepj**
Rails 5.2's EOL is 1.6.2022 so it would be nice to remove `ActiveRecord::Import` and replace it for Rails 6 bulk insert in the next RES. Yes, it's a breaking change so it maybe suitable for RES 3.x.

Tests passed locally (`make test`).

### CODE REVIEW

**Comment by pawelpacana** [comment]
In fact this change was discussed internally and will be welcomed addition to breaking 3.x release :+1:

File: contrib/ruby_event_store-outbox/Gemfile.sidekiq_5_2 [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-
-
-gem 'ruby_event_store', path: '../../ruby_event_store'
-gem 'sidekiq', '5.2.9'
-gem 'pg', '1.2.2'
-gem 'mysql2', '0.5.3'
-gem 'influxdb', '~> 0.8.1', require: false
-gem 'childprocess'
-gem 'rails', '6.1.0'
-gem 'sqlite3', '1.4.2'
```
Comment (pawelpacana) [inline contrib/ruby_event_store-outbox/Gemfile.sidekiq_5_2:14]: Perhaps one gemfile too far?

Comment (deepj) [reply contrib/ruby_event_store-outbox/Gemfile.sidekiq_5_2]: Restored

**Comment by pawelpacana** [comment]
> @pawelpacana haven't we agreed that removing support for EOLed rubies/rails won't need a major release? In that case we could merge that PR just after rails gets EOLed

We did, we may want to release 3.0 before 5.2 gets EOLed.

**Pull request closed without merging**